spec/formalism/form/coercion_spec.rb

```diff
@@ -0,0 +1,189 @@
+# frozen_string_literal: true
+
+## This is unit test, created to simplify testing of different coercions
+## You can find integration test in spec/formalism/form/fields_spec.rb
+## (Formalism::Form -> .field -> coersion)
+##
+## In tests `eql` used instead of `eq` to determine result type
+describe Formalism.const_get(:Coercion) do
+	let(:value) {}
+	let(:type) {}
```

Comment:
For what?

Reply:
Removed it. I think it is nice to define `let`s in context where it used, but it is not common